Kitodo-DataManagement/src/main/resources/db/migration/V2_62__Rename_the_table_folder_to_subfolder_type.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Rename the table folder to subfolder type
+--
+-- 62. Rename the table folder to subfolder type
+--
+
+USE kitodo;
```

Comment:
You can not assume that the database name is `Kitodo` and until now this `use` statement is not used in previous migrations.